spec/unit/shoulda/matchers/active_model/validate_numericality_of_matcher_spec.rb

```diff
@@ -1141,6 +1216,21 @@ def define_model_validating_numericality(options = {})
     end
   end
 
+  def define_model_validating_numericality_of_virtual_attribute(options = {})
+    virtual_attr = options.delete(:virtual_attribute) do
+      virtual_attribute_name
+    end
+
+    define_model 'Example' do |model|
```

Comment:
I realize that's not what your use case is, but the numericality matcher should work for both ActiveRecord and ActiveModel models, and we don't have tests for ActiveModel here I'm noticing. So we could kill two birds with one stone. What do you think?